packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -616,6 +616,48 @@ class _InkResponseState<T extends InkResponse> extends State<T> with AutomaticKe
 /// ancestor to the ink well). The [MaterialType.transparency] material
 /// kind can be used for this purpose.
 ///
+/// ### The ink splashes don't track the size of an animated container
+/// When you wrap an animating widget with an InkWell, you may notice that ink
```

Comment:
If the size of an InkWell's Material ancestor changes while the InkWell's splashes are expanding, you may notice that the splashes aren't clipped correctly. This can't be avoided.